### PR TITLE
fix(issue-stream): Fix truncated title

### PR DESCRIPTION
**Pull request by priscilawebdev**
**Before:**

image

**After:**

image

closes: ISSUE-1286

### CODE REVIEW

File: static/app/components/eventOrGroupHeader.tsx [hunk 1]
```diff
@@ -104,38 +108,37 @@ class EventOrGroupHeader extends Component<Props> {
               eventID ? `events/${eventID}/` : ''
             }`,
             query: {
-              query: this.props.query,
-              ...(location.query.sort !== undefined ? {sort: location.query.sort} : {}), // This adds sort to the query if one was selected from the issues list page
-              ...(location.query.project !== undefined ? {} : {_allp: 1}), // This appends _allp to the URL parameters if they have no project selected ("all" projects included in results). This is so that when we enter the issue details page and lock them to a project, we can properly take them back to the issue list page with no project selected (and not the locked project selected)
+              query,
+              ...(loc.query.sort !== undefined ? {sort: loc.query.sort} : {}), // This adds sort to the query if one was selected from the issues list page
```
Comment (matejminar) [inline static/app/components/eventOrGroupHeader.tsx:112]: Should we find a better alias name for `location`? My mind jumped to `Lines Of Code`.

**Review by matejminar — APPROVED** [review]
image
🤷‍♂️

File: static/app/components/eventOrGroupHeader.tsx (hunk 1 above)
Comment (priscilawebdev) [reply static/app/components/eventOrGroupHeader.tsx:112]: location is already being used and I did not want to change that.. but I can use `props.location `

File: tests/js/spec/components/eventOrGroupHeader.spec.jsx [hunk 2]
```diff
@@ -191,7 +191,7 @@ describe('EventOrGroupHeader', function () {
         sort: 'freq',
       };
 
-      const component = shallow(
```
Comment (scttcper) [inline tests/js/spec/components/eventOrGroupHeader.spec.jsx:194]: nice!

**Pull request merged**